docs/source/ngv.rst

```diff
@@ -28,5 +28,87 @@ The diagram also illustrates the various connections among the populations, as d
+
+Astrocyte Structure and Dynamics
+--------------------------------
+
+Each astrocyte is composed of multiple sections:
+
+- **Soma section**:
+
+  - Contains three key components:
+
+    - A `glutReceive` point process.
+    - A `glutReceiveSoma` point process, which monitors recent synaptic activity across the entire astrocyte by counting signals received within the last millisecond. If no signals arrive during this period, the counter is reset, enabling real-time responsiveness.
+    - A `cadifus` mechanism responsible for calcium diffusion, configured to observe both `glutReceive` and `glutReceiveSoma`.
+
+  Due to the order of section initialization, the `cadifus` pointer is overwritten by the `glutReceiveSoma` reference when the soma is connected last. Consequently, the `glutReceive` in the soma is likely unused in practice.
+
+- **Endfeet sections**:
+
+  - Instantiated during astrocyte creation.
+  - Each contains:
+  
+    - A ``glutReceive`` point process that counts all received glutamate signals.
+    - A ``cadifus`` mechanism that diffuses calcium based on this counter and resets the glutamate count.
+    - A ``vascouplingB`` mechanism that modulates the blood vessel radius in response to local calcium concentration.
+
+  - Each endfoot is instantiated as a new neuron section and added to the ``endfeet`` SectionList. They are not connected to each other but are connected to the ``source_node_id``.
+
+    Contrary to the `documentation <https://sonata-extension.readthedocs.io/en/latest/sonata_tech.html#fields-for-endfoot-connection-type-edges>`_, the fields ``vasculature_section_id`` and ``vasculature_segment_id`` are not mandatory. In practice, they are unused by both the SONATA reader and Neurodamus.
+
+    Only the ``source_node_id`` field is required, as the ``section_id`` and ``segment_id`` can be used to infer it. Including these additional fields may introduce inconsistencies.
+
+Signal Processing and Vascular Modulation Chain
+-----------------------------------------------
+
+The sequence of interactions leading to blood vessel modulation is as follows:
+
+1. A presynaptic neuron fires and sends a signal to the spike exchange.
+2. The targeted synapse processes this signal through its `NET_RECEIVE` block and may fire.
+3. If the synapse is part of a tripartite connection and it fires, `Ustate` is set to 1.
+4. This event sends another signal to the spike exchange, which is collected by `glutReceive` and `glutReceiveSoma` point processes on the astrocyte.
```

Comment:
minor comment regarding the svg - the UState has an arrow going to a different section than the one holding the synapse. Maybe it is difficult to avoid having so many arrows in the same space?

Reply:
done